rory.gemspec

```diff
@@ -35,11 +35,12 @@ EOF
 
   s.add_development_dependency 'rake', '~> 10.4'
   s.add_development_dependency 'rspec', '~> 3'
+  s.add_development_dependency 'mime-types', '~> 2.6'
```

Comment:
Curious about this - why do we have to explicitly call out this dependency?  Doesn't Capybara handle that?  Is there a version conflict?